test/e2e/test-artifacts/generated-test-artifacts/standard-cluster-create/cluster.json

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
a) if generated, why checked in?
b) why not the .bicep? much more ergonomic to do anything with, if this is actually the source of truth for something, much nicer to edit the .bicep and transpile on the fly